src/bin/common.rs

```diff
@@ -389,7 +389,7 @@ impl ProgressInfo {
     Intra Only: {:>6}    avg size: {:>7} B\n\
     Switch:     {:>6}    avg size: {:>7} B",
       key, key_size / key,
-      inter, inter_size / inter,
+      inter, if inter == 0 { 0 } else { inter_size / inter },
```

Comment:
```suggestion
      inter, inter_size.checked_div(inter).ok_or(0),
```

Maybe?

or we could directly not print the line?